docs/source/manual/file_formats/bitstream_setting.rst

```diff
@@ -16,6 +16,7 @@ This can define a hard-coded bitstream for a reconfigurable resource in FPGA fab
     <non_fabric name="<string>" file="<string>">
       <pb name="<string>" type="<string>" content="<string>"/>
     </non_fabric>
+    <bit value="<0 or 1>" path="<string>"/>
```

Comment:
please use "string" for value, as in other configuration protocol, "x" may be occur

Reply:
This is archiecture bitstream (not configuration protocol dependent bitstream: for example scan chain or ql_memory_bank or etc which might have don't care).

See how we add the bit to architecture bitstream, we only accept boolean (two possible: true or false, which we only write '1' or '0') - there is no dont care - https://github.com/lnis-uofu/OpenFPGA/blob/master/libs/libfpgabitstream/src/bitstream_manager.cpp#L156-L170

See how we read the bit in architecture bitstream, we only return boolean (two possible: true or false) - https://github.com/lnis-uofu/OpenFPGA/blob/master/libs/libfpgabitstream/src/bitstream_manager.cpp#L48-L53

So I do not see any reason to support "x" dont care bit in architecture bitstream setting based on our current code.

I do not plan to change that unless you have a valid reason to insist.

Reply:
Make sense to me. Architecture bitstream does not need 'x'